spring-jdbc/src/main/java/org/springframework/jdbc/datasource/init/DatabasePopulatorUtils.java

```diff
@@ -52,10 +52,10 @@ public static void execute(DatabasePopulator populator, DataSource dataSource) t
 				DataSourceUtils.releaseConnection(connection, dataSource);
 			}
 		}
+		catch (ScriptException ex){
```

Comment:
Please note that the above is missing a space: `) {`, but I'll fix that after merging.